spec/models/user_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  user = User.create(name: 'Ana', photo: 'https://photoUrl.com', bio: 'Teacher from Mexico.')
+
+  context 'User is created with proper key and values, passes all validations and it ' do
+    it 'should have a key name = Ana' do
+      expect(user.name).to match('Ana')
+    end
+    it 'should have a key photo = https://photoUrl.com' do
+      expect(user.photo).to match('https://photoUrl.com')
+    end
+    it 'should have a key bio = Teacher from Mexico' do
+      expect(user.bio).to match('Teacher from Mexico')
+    end
+    it 'should have a key posts_counter = 0' do
+      expect(user.posts_counter).to match(0)
+    end
```

Comment:
- [ ] Kindly add a test case to check for the posts counter to be an integer greater than or equal to 0, as this is a requirement. Currently it only checks if it is matches 0

Reply:
I've done it!

Thanks for pointing that out!